### PR TITLE
SOC-2492 Add hero and badge images

### PR DESCRIPTION
**Pull request by garthwebb**
Add images for Japanese overwatch for https://wikia-inc.atlassian.net/browse/SOC-2492

These images just point to existing English Overwatch assets.

### CODE REVIEW

File: server/config/discussionsSplashPageConfig.js [hunk 1]
```diff
@@ -448,4 +448,11 @@ export default {
 		language: 'ja',
 		dbName: 'jaessandboxsidonia'
 	},
+
```
Comment (kvas-damian) [inline server/config/discussionsSplashPageConfig.js]: You don't need to update this file until you want to specify mobile app link.

Comment (kvas-damian) [inline server/config/discussionsSplashPageConfig.js]: BTW this section violates linter rules ;)

Comment (garthwebb) [inline server/config/discussionsSplashPageConfig.js]: Huh, I wonder why commit didn't complain ...

**Comment by bkoval** [comment]
LGTM 👍

**Comment by kvas-damian** [comment]
Lgtm

**Pull request merged**